xml/admin_ceph_tips.xml

```diff
@@ -367,8 +367,83 @@ osd_scrub_load_threshold = 0.25
    </sect3>
   </sect2>
 
+  <sect2 xml:id="storage-tips-ceph-btrfs-subvol-automatic">
+   <title>Steps Required during a New Cluster Deployment</title>
+   <sect3 xml:id="var-lib-ceph-stage0">
+    <title>Before Running &deepsea; stage.0</title>
+    <para>
+     Prior to running &deepsea; stage.0, apply the following commands to each
+     of the &sminion;s that will become &mon;s:
+    </para>
+<screen>
+&prompt.smaster;salt '<replaceable>MONITOR_NODES</replaceable>' saltutil.sync_all
+&prompt.smaster;salt '<replaceable>MONITOR_NODES</replaceable>' state.apply ceph.subvolume
+</screen>
+    <para>
+     The <command>ceph.subvolume</command> command does the following:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       Creates <filename>/var/lib/ceph</filename> as a
+       <literal>@/var/lib/ceph</literal> Btrfs subvolume.
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       Mount the new subvolume and update <filename>/etc/fstab</filename>
+       appropriately.
```

Comment:
Shouldn't this be "Mounts" and "updates" to match "Creates"?

Reply:
yep, fixed